inc/namespace.php

```diff
@@ -221,20 +221,42 @@ function enqueue_scripts() {
 	 */
 	$noop = (bool) apply_filters( 'altis.analytics.noop', false );
 
+	/**
+	 * Filters the consent cookie prefix to integrate with the WordPress Consent API.
+	 *
+	 * If the value is `null` then no special consent handling is done and Analytics will
+	 * run as normal.
+	 *
+	 * @param string|null $cookie_prefix The consent cookie prefix or false to skip consent check.
+	 */
+	$consent_cookie_prefix = apply_filters( 'altis.analytics.consent_cookie_prefix', null );
+
+	/**
+	 * Filters the consent category required to run analytics.
+	 *
+	 * @param string $consent_category The consent category to allow analytics to run with.
+	 */
+	$consent_category = apply_filters( 'altis.analytics.consent_category', 'statistics' );
```

Comment:
I'm happy with this being filterable externally. And maybe, as you say, we just call it statistics whether it's user-identifying information or not. By default, in the consent plugin, I'm assuming that anonymous statistics are always allowed through, and am bypassing requesting consent for those purposes, so, even if the data itself isn't identifying, it may be better to just ask by default. 

However, I would expect that, if we are doing some anonymous or non-personally-identifiable data tracking, that there would also be something adding a filter in here or modifying this filter to use the anonymous-statistics category, instead. See also https://github.com/humanmade/aws-analytics/pull/131#discussion_r546012126

Reply:
Just went for hardcoding these in the end. Unlikely they'll change and we're only integrating with the WP consent API here.